crates/aptos-rosetta-cli/README.md

```diff
@@ -0,0 +1,7 @@
+## Rosetta CLI
+
+The Rosetta CLI is used for testing, and is not used for any production use case.  The CLI uses the same client as the end to end tests to ensure consistency, but with the caveat that it will panicc for cases that aren't expected.
+
```

Comment:
panicc → panic

Reply:
I fixed this while making some other edits in a commit here.